### ISSUE TITLE
question about code

### ISSUE THREAD

**Issue by supersonicMaclaurin** [XiangFeng66/SRGS#10]
If I want to reproduce the experimental results under x4, how should I set the - r for train and render? I tried using train-r 8 to train the Synthetic NeRF Dataset chair scene (original image size is 804 * 804), and during the camera loading process, there was an output size of 400 * 400. Afterwards, when rendering, the output size using render.by - r 1 is 3216 * 3216, the output size using render.by - r 4 is 804 * 804, and the output size using render.by - r 8 is 400 * 400

**Comment by XiangFeng66** [comment]
@supersonicMaclaurin Hello, thank you for your attention to SRGS. According to your description, r is the multiple you want to upsample and downsample the original dataset, that is, the process of obtaining LR. SRGS will automatically perform x4 upsampling. The rendering process should be consistent with the training process.

**Comment by supersonicMaclaurin** [comment]
So if  I want to get x4 results, I should run 
```
train.py -r 4
render.py -r 4
```
Is it correct ?

**Comment by XiangFeng66** [comment]
If you want to go from 200 to 800, you should do this. If you want to go from 100 to 400, -r should be 8

**Comment by supersonicMaclaurin** [comment]
Thank you for your prompt response. I have some questions about data processing. I noticed in the issue (https://github.com/XiangFeng66/SRGS/issues/4) that you mentioned using code from 3DGS for downsampling processing. I would like to know if I should first downsample the raw data, then use COLMAP to process the LR image, and then execute train.exe; Or is it that the downsampling code is already included in the code https://github.com/XiangFeng66/SRGS/blob/a80754ffeb9ca6a905efb7141b1a801f3f16d16e/utils/camera_utils.py#L23 ,so there is no need to manually prepare downsampling images in advance

**Comment by supersonicMaclaurin** [comment]
Additionally, I have another question. I attempted to replicate using the Synthetic NeRF dataset, which was pre divided into training and testing sets. 
```
nerf_synthetic
    | train (100images)
    | test (200images)
    | val
```
I processed each set separately using COLMAP and get processed results like
```
nerf_synthetic_colmap
    | train 
        | images
        | sparse
            | 0  (cameras.bin, images.bin, points3D.bin)
    | test (200images)
        | images
        | sparse
            | 0  (cameras.bin, images.bin, points3D.bin)
``` 
ans then run 
```
python train.py -s Dataset/nerf_synthetic_colmap/chair/train  --model_path output/chair_r4 -r 4
```
After training with the training set images, I specified the - s parameter as the path of the test set, hoping to render the results for metric evaluation, but the results were not very good. 
```
python render.py -m output/chair_r4  -r 4 -s Dataset/nerf_synthetic_colmap/chair/test 
``` 
I would like to know if there is any problem with my operation ?
The rendered images almost always contain only incomplete parts of the scene 

![Image](https://github.com/user-attachments/assets/2faa6548-650f-4c0a-bef3-1f060b7afeed)

**Comment by XiangFeng66** [comment]
@supersonicMaclaurin 
First, I use the data downsampling code in the 3DGS source code. This setting is consistent with Mip-splatting. Second, the blender dataset does not need to use colmap. 3DGS has a corresponding reading method. You should look at the 3DGS source code.
